src/cantools/database/can/formats/dbc.py

```diff
@@ -1575,48 +1591,37 @@ def get_comment(frame_id_dbc):
             return comments[frame_id_dbc]['message']
         except KeyError:
             return None
-
-    def get_send_type(frame_id_dbc):
-        """Get send type for a given message.
-
+        
+    def get_attribute_value(frame_id_dbc, attribute):
+        """Get the attribute value of a given frame id. Currently supported attributes:
+           - GenMsgSendType: specifies the whether a message is cyclic or sporadic
+           - Type: It is a further specification w.r.t. to GenMsgSendType that gives more details on whether the message is cyclic, sporadic, mixed, etc.
+           - GenMsgCycleTime: Defines the period of a cyclic message
+           - GenMsgDelayTime: Defines the offset w.r.t. to the cycle time
+           - GenMsgStartDelayTime: Defines the time interval between start-up and the first message
         """
```

Comment:
Seems like other attributes may be supported as well.
Maybe smth like:

```suggestion
        """Get the value of an attribute for a given frame id, possibly returning a default value if it exsists and the attribute is not available, without any ad-hoc processing.
        """
```